src/db/file.rs

```diff
@@ -225,7 +224,52 @@ fn file_list_to_json(file_list: Vec<(String, PathBuf)>) -> Result<Json> {
     Ok(file_list_json.to_json())
 }
 
+pub fn move_to_s3(conn: &Connection, n: usize) -> Result<()> {
+    let trans = try!(conn.transaction());
+    let client = s3_client().expect("configured s3");
+
+    let rows = try!(trans.query(
+            &format!("SELECT path, mime, content FROM files WHERE content != E'in-s3' LIMIT {}", n),
+            &[]));
+
+    let mut rt = ::tokio::runtime::current_thread::Runtime::new().unwrap();
```

Comment:
We use a single-threaded runtime here because something in docs.rs (I think, maybe postgres or something) is using a `Cell` which can't be shared/sent between threads. I don't think it matters much -- maybe slows us down, but that seems fine.

Reply:
Since the futures are probably going to be mostly IO-bound anyway, using the single-threaded runtime seems fine to me too.